namespaces/live.cloud-platform.service.justice.gov.uk/laa-ccms-staging-file-upload-api-production/resources/ecr.tf

```diff
@@ -0,0 +1,29 @@
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "ecr" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=7.0.0"
+
+  # Repository configuration
+  team_name = var.team_name
+  repo_name = "${var.namespace}-ecr"
+  namespace = var.namespace
+
+  # OpenID Connect configuration
+  oidc_providers        = ["github"]
+  github_repositories   = ["laa-ccms-staging-file-upload-api"]
+  github_actions_prefix = "production"
+
+  # OPTIONAL: GitHub environments, to create variables as actions variables in your environments
+  github_environments = ["production"]
```

Comment:
Double check if we have **production** environment in github repo.

Reply:
yes we have a prod and production env